src/transformers/models/roberta/modeling_tf_roberta.py

```diff
@@ -704,6 +704,8 @@ def call(
             extended_attention_mask = tf.reshape(
                 extended_attention_mask, (attention_mask_shape[0], 1, attention_mask_shape[1], attention_mask_shape[2])
             )
+            if inputs["past_key_values"][0] is not None:
```

Comment:
Good catch!

Reply:
Could you maybe add a comment here
```suggestion
            if inputs["past_key_values"][0] is not None:
                # attention_mask needs to be sliced to the shape `[batch_size, 1, from_seq_length - cached_seq_length, to_seq_length]
```

Reply:
Same for all other models

Reply:
OK - I will also add the same comment to other models with this block